skt/reporter.py

```diff
@@ -65,7 +65,12 @@ def load_state_cfg(statefile):
     """
     cfg = {}
     state_to_report = ConfigParser.ConfigParser()
-    state_to_report.read(statefile)
+    read_file = state_to_report.read(statefile)
+    if not read_file:
+        raise OSError(
+            2,
+            'Can\'t create report for %s, file not found!' % statefile
+        )
```

Comment:
This would still fire, if we e.g. didn't have permission to read the file (common thing when messing around with Jenkins users), and the error would be confusing.

Could you use `open()` and `state_to_report.readfp()` instead and produce appropriate error messages, or exceptions?